packages/deployer/utils/tests.js

```diff
@@ -13,19 +13,20 @@ function bootstrap(initializer = () => {}, customDeployOptions = {}) {
     instance: 'test',
   };
 
-  before('take a snapshot', async () => {
+  before('take a snapshot', async function () {
     snapshotId = await takeSnapshot(hre.ethers.provider);
   });
 
-  before('deploy system', async () => {
+  before('deploy system', async function () {
+    this.timeout(100000);
```

Comment:
Added the timeout since the deployment takes more than 20s, at least in my local machine.